metrics/group_processes.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2022 Tigris Data, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"github.com/rs/zerolog/log"
+	"github.com/tigrisdata/fdb-exporter/models"
+)
+
+type ProcessesMetricGroup struct {
+	metricGroup
+}
+
+func NewProcessesMetricGroup(reporter *MetricReporter) *ProcessesMetricGroup {
+	return &ProcessesMetricGroup{*newMetricGroup("processes", reporter.GetScopeOrExit("cluster"), reporter)}
+}
+
+func (p *ProcessesMetricGroup) getTags(processName string, process *models.Process) map[string]string {
+	tags := GetBaseTags()
+	tags["name"] = processName
+	tags["class_type"] = process.ClassType
+	// On the same tally scope the same tag set should be present
+	tags["storage"] = "0"
```

Comment:
Consider exposing the meaning of "0" and "1" for these contexts. How does `tags["storage"] = "1"` impact other things?